tests/tests-downloads.php

```diff
@@ -133,7 +132,7 @@ public function test_edd_download() {
 		$this->assertEquals( $prices, $download3->prices );
 		$this->assertEquals( $prices, $download3->get_prices() );
 		$this->assertEquals( 6, $download3->sales );
-		$this->assertEquals( 120.50, $download3->earnings );
+		$this->assertEquals( 120.00, $download3->earnings );
```

Comment:
All download ledger tests are removed--those functions/methods have been updated to run recalcuations if called, but the incrementing is ignored. But doing things like `$download3->increase_earnings( '0.50' );` is now nonsensical and shouldn't actually affect the earnings, because it's an artificial adjustment.